### DISCUSSION TITLE
What happened to io_uring_spawn

### DISCUSSION THREAD

**Discussion by Asmod4n** [axboe/liburing#1307] [General]
Hi, i've stumbled about a talk from 2020 which said io_uring_spawn is just around the corner as a faster alternative to vfork, what happened to it?

**Comment by axboe** [comment]
https://lore.kernel.org/io-uring/1ae79f05-1a07-40aa-acf7-8af98b14b94f@gmail.com/T/#m18a7d5141d656eb045c3b2326828b1179a3b72d2
